Update framework config (#18773)
symfony/symfony-docs: reference/configuration/framework.rst

handle_all_throwables
~~~~~~~~~~~~~~~~~~~~~

**type**: ``boolean`` **default**: ``true``

> Set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions

Reviewer: ```suggestion
When at ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
```

Maybe?

> When at ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions

Reviewer 2: ```suggestion
When set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
```

If set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
When set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
thrown by the application and will turn them into HTTP responses.

> Starting from Symfony 7.0, the default value of this option will be ``true``.

Reviewer 3: Did you mean to target the `7.0` branch?

Author: Yeah, and yes my target branch wasn't the right one ;)

handler_id
..........

**type**: ``string`` | ``null`` **default**: ``null``

> By default at `null` if `framework.session.save_path` is not set.

Reviewer 2: ```suggestion
By default at ``null`` if ``framework.session.save_path`` is not set.
```

> The service id used for session storage. The default value ``'session.handler.native_file'``

Reviewer 4: By default, the session handler configured by php.ini is used, unless option save_path is used, in which case the default is to store sessions using the native file session handler.

The session handler configured by php.ini is used, unless option

> `framework.session.save_path` is used, in which case the default is to store sessions

Reviewer 2: ```suggestion
``framework.session.save_path`` is used, in which case the default is to store sessions
```

save_path
.........

> **type**: ``string`` or ``null`` **default**: ``%kernel.cache_dir%/sessions``

Reviewer: ```suggestion
**type**: ``string`` | ``null`` **default**: ``%kernel.cache_dir%/sessions``
```

log
...

> **type**: ``boolean|int`` **default**: ``true``

Reviewer: ```suggestion
**type**: ``boolean`` | ``int`` **default**: ``true``
```
